Cachegroup parameters API tests fail

<!--
************ STOP!! ************
If this issue identifies a security vulnerability, DO NOT submit it! Instead, contact
the Apache Software Foundation Security Team at security@trafficcontrol.apache.org and follow the
guidelines at https://www.apache.org/security/ regarding vulnerability disclosure.
-->

<!--
- For *SUPPORT QUESTIONS*, use the
[Traffic Control slack channels](https://traffic-control-cdn.slack.com) or [Traffic Control mailing lists](http://trafficcontrol.apache.org/mailing_lists/).
- Before submitting, please **SEARCH GITHUB** for a similar issue or PR. -->

## I'm submitting a ...
<!-- (check all that apply with "[x]") -->
<!--- security vulnerability (STOP!! - see above)-->
- [x] bug report
- [ ] new feature / enhancement request
- [ ] improvement request (usability, performance, tech debt, etc.)
- [ ] other <!--(Please do not submit support requests here - see above)-->

## Traffic Control components affected ...
<!-- (check all that apply with "[x]") -->
- [ ] CDN in a Box
- [ ] Documentation
- [ ] Grove
- [ ] Traffic Control Client
- [ ] Traffic Monitor
- [x] Traffic Ops
- [ ] Traffic Ops ORT
- [ ] Traffic Portal
- [ ] Traffic Router
- [ ] Traffic Stats
- [ ] Traffic Vault
- [ ] unknown

## Current behavior:
<!-- Describe how the bug manifests / how the current features are insufficient. -->
Cachegroup parameters API tests fail at fd838becca

## Expected / new behavior:
<!-- Describe what the behavior would be without the bug / how the feature would improve Traffic Control -->
Cachegroup parameters API tests should succeed

## Minimal reproduction of the problem with instructions:
<!--
If the current behavior is a bug or you can illustrate your feature request better with an example,
please provide the *STEPS TO REPRODUCE* and include the applicable TC version.
-->
1. Start CDN-in-a-Box with exposed ports and wait Traffic Ops Perl to finish seeding the database
    ```shell script
    docker-compose -f docker-compose.yml -f docker-compose.expose-ports.yml up -d
    docker-compose -f docker-compose.yml -f docker-compose.expose-ports.yml logs -f trafficops-perl
    ```

1. Configure `traffic_ops/testing/api/conf/traffic-ops-test.conf` for CDN-in-a-Box
    ```json
    {
        "default": {
            "logLocations": {
                "debug": "stdout",
                "error": "stdout",
                "event": "stdout",
                "info": "stdout",
                "warning": "stdout"
            },
            "session": {
                "timeoutInSecs": 60
            },
            "includeSystemTests": false
        },
        "use_ims": true,
        "trafficOps": {
            "URL": "https://localhost:6443",
            "password": "twelve",
            "users": {
                "disallowed": "disallowed",
                "operations": "operations",
                "admin": "admin",
                "federation": "federation",
                "portal": "portal",
                "readOnly": "readOnly",
                "extension": "extension"
            }
        },
        "trafficOpsDB": {
            "dbname": "traffic_ops",
            "description": "Test database traffic_ops",
            "hostname": "localhost",
            "password": "twelve",
            "port": "5432",
            "type": "Pg",
            "user": "traffic_ops"
        }
    }
    ```

1. Run the cachegroup parameters API tests
    ```shell script
    go test ./v3 -test.v -cfg=../conf/traffic-ops-test.conf -fixtures=tc-fixtures.json -test.run 'TestMain|TestCacheGroupParameters'
    ```

## Anything else:
<!-- e.g. stacktraces, related issues, suggestions how to fix -->
Cachegroup API tests log:
```
INFO: traffic_ops_test.go:71: 2020-06-30T14:10:46.528087208-06:00: Using Config values:
			   TO Config File:       ../conf/traffic-ops-test.conf
			   TO Fixtures:          tc-fixtures.json
			   TO URL:               https://localhost:6443
			   TO Session Timeout In Secs:  60
			   DB Server:            localhost
			   DB User:              traffic_ops
			   DB Name:              traffic_ops
			   DB Ssl:               false
               UseIMS:               true
=== RUN   TestCacheGroupParameters
    TestCacheGroupParameters: types_test.go:87: ---- CreateTestTypes ----
    TestCacheGroupParameters: parameters_test.go:75: Response:  {[{param was created. success}]}
    TestCacheGroupParameters: parameters_test.go:75: Response:  {[{param was created. success}]}
    TestCacheGroupParameters: parameters_test.go:75: Response:  {[{param was created. success}]}
    TestCacheGroupParameters: parameters_test.go:75: Response:  {[{param was created. success}]}
    TestCacheGroupParameters: parameters_test.go:75: Response:  {[{param was created. success}]}
    TestCacheGroupParameters: parameters_test.go:75: Response:  {[{param was created. success}]}
    TestCacheGroupParameters: parameters_test.go:75: Response:  {[{param was created. success}]}
    TestCacheGroupParameters: parameters_test.go:75: Response:  {[{param was created. success}]}
    TestCacheGroupParameters: parameters_test.go:75: Response:  {[{param was created. success}]}
    TestCacheGroupParameters: parameters_test.go:75: Response:  {[{param was created. success}]}
    TestCacheGroupParameters: parameters_test.go:75: Response:  {[{param was created. success}]}
    TestCacheGroupParameters: parameters_test.go:75: Response:  {[{param was created. success}]}
    TestCacheGroupParameters: parameters_test.go:75: Response:  {[{param was created. success}]}
    TestCacheGroupParameters: parameters_test.go:75: Response:  {[{param was created. success}]}
    TestCacheGroupParameters: parameters_test.go:75: Response:  {[{param was created. success}]}
    TestCacheGroupParameters: parameters_test.go:75: Response:  {[{param was created. success}]}
    TestCacheGroupParameters: parameters_test.go:75: Response:  {[{param was created. success}]}
    TestCacheGroupParameters: parameters_test.go:75: Response:  {[{param was created. success}]}
    TestCacheGroupParameters: parameters_test.go:75: Response:  {[{param was created. success}]}
    TestCacheGroupParameters: parameters_test.go:75: Response:  {[{param was created. success}]}
    TestCacheGroupParameters: cachegroups_parameters_test.go:94: Expected 304 status code, got 200
    TestCacheGroupParameters: types_test.go:187: ---- DeleteTestTypes ----
--- FAIL: TestCacheGroupParameters (2.01s)
FAIL
FAIL	github.com/apache/trafficcontrol/traffic_ops/testing/api/v3	2.560s
FAIL
```

<!--
Licensed to the Apache Software Foundation (ASF) under one
or more contributor license agreements.  See the NOTICE file
distributed with this work for additional information
regarding copyright ownership.  The ASF licenses this file
to you under the Apache License, Version 2.0 (the
"License"); you may not use this file except in compliance
with the License.  You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing,
software distributed under the License is distributed on an
"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
KIND, either express or implied.  See the License for the
specific language governing permissions and limitations
under the License.
-->

--- a/infrastructure/cdn-in-a-box/traffic_ops/config.sh
+++ b/infrastructure/cdn-in-a-box/traffic_ops/config.sh
@@ -83,6 +83,7 @@ cat <<-EOF >/opt/traffic_ops/app/conf/cdn.conf
         "pid_file" : "/var/run/traffic_ops.pid",
         "workers" : 12
     },
+    "use_ims": true,
     "traffic_ops_golang" : {
         "insecure": true,
         "port" : "$TO_PORT",

--- a/traffic_ops/testing/api/v3/cachegroups_test.go
+++ b/traffic_ops/testing/api/v3/cachegroups_test.go
@@ -37,7 +37,7 @@ func TestCacheGroups(t *testing.T) {
 		GetTestCacheGroupsByShortName(t)
 		GetTestCacheGroupsByTopology(t)
 		CheckCacheGroupsAuthentication(t)
-		currentTime := time.Now().Add(-1 * time.Second)
+		currentTime := time.Now().UTC().Add(-5 * time.Second)
 		time := currentTime.Format(time.RFC1123)
 		var header http.Header
 		header = make(map[string][]string)
@@ -55,7 +55,7 @@ func GetTestCacheGroupsAfterChangeIMS(t *testing.T, header http.Header) {
 	if reqInf.StatusCode != http.StatusOK {
 		t.Fatalf("Expected 200 status code, got %v", reqInf.StatusCode)
 	}
-	currentTime := time.Now()
+	currentTime := time.Now().UTC()
 	currentTime = currentTime.Add(1 * time.Second)
 	timeStr := currentTime.Format(time.RFC1123)
 	header.Set(rfc.IfModifiedSince, timeStr)

--- a/traffic_ops/testing/api/v3/cdns_test.go
+++ b/traffic_ops/testing/api/v3/cdns_test.go
@@ -27,7 +27,7 @@ import (
 func TestCDNs(t *testing.T) {
 	WithObjs(t, []TCObj{CDNs, Parameters}, func() {
 		GetTestCDNsIMS(t)
-		currentTime := time.Now().Add(-1 * time.Second)
+		currentTime := time.Now().UTC().Add(-5 * time.Second)
 		time := currentTime.Format(time.RFC1123)
 		var header http.Header
 		header = make(map[string][]string)
@@ -48,7 +48,7 @@ func GetTestCDNsIMSAfterChange(t *testing.T, header http.Header) {
 			t.Fatalf("Expected 304 status code, got %v", reqInf.StatusCode)
 		}
 	}
-	currentTime := time.Now()
+	currentTime := time.Now().UTC()
 	currentTime = currentTime.Add(1 * time.Second)
 	timeStr := currentTime.Format(time.RFC1123)
 	header.Set(rfc.IfModifiedSince, timeStr)

--- a/traffic_ops/testing/api/v3/coordinates_test.go
+++ b/traffic_ops/testing/api/v3/coordinates_test.go
@@ -28,7 +28,7 @@ func TestCoordinates(t *testing.T) {
 	WithObjs(t, []TCObj{Parameters, Coordinates}, func() {
 		GetTestCoordinatesIMS(t)
 		GetTestCoordinates(t)
-		currentTime := time.Now().Add(-1 * time.Second)
+		currentTime := time.Now().UTC().Add(-5 * time.Second)
 		time := currentTime.Format(time.RFC1123)
 		var header http.Header
 		header = make(map[string][]string)
@@ -48,7 +48,7 @@ func GetTestCoordinatesIMSAfterChange(t *testing.T, header http.Header) {
 			t.Fatalf("Expected 304 status code, got %v", reqInf.StatusCode)
 		}
 	}
-	currentTime := time.Now()
+	currentTime := time.Now().UTC()
 	currentTime = currentTime.Add(1 * time.Second)
 	timeStr := currentTime.Format(time.RFC1123)
 	header.Set(rfc.IfModifiedSince, timeStr)

--- a/traffic_ops/testing/api/v3/deliveryservice_request_comments_test.go
+++ b/traffic_ops/testing/api/v3/deliveryservice_request_comments_test.go
@@ -27,7 +27,7 @@ import (
 func TestDeliveryServiceRequestComments(t *testing.T) {
 	WithObjs(t, []TCObj{CDNs, Types, Parameters, Tenants, DeliveryServiceRequests, DeliveryServiceRequestComments}, func() {
 		GetTestDeliveryServiceRequestCommentsIMS(t)
-		currentTime := time.Now().Add(-1 * time.Second)
+		currentTime := time.Now().UTC().Add(-5 * time.Second)
 		time := currentTime.Format(time.RFC1123)
 		var header http.Header
 		header = make(map[string][]string)

--- a/traffic_ops/testing/api/v3/deliveryservice_requests_test.go
+++ b/traffic_ops/testing/api/v3/deliveryservice_requests_test.go
@@ -37,7 +37,7 @@ func TestDeliveryServiceRequests(t *testing.T) {
 	WithObjs(t, []TCObj{CDNs, Types, Parameters, Tenants, DeliveryServiceRequests}, func() {
 		GetTestDeliveryServiceRequestsIMS(t)
 		GetTestDeliveryServiceRequests(t)
-		currentTime := time.Now().Add(-1 * time.Second)
+		currentTime := time.Now().UTC().Add(-5 * time.Second)
 		time := currentTime.Format(time.RFC1123)
 		var header http.Header
 		header = make(map[string][]string)
@@ -56,7 +56,7 @@ func GetTestDeliveryServiceRequestsIMSAfterChange(t *testing.T, header http.Head
 	if reqInf.StatusCode != http.StatusOK {
 		t.Fatalf("Expected 200 status code, got %v", reqInf.StatusCode)
 	}
-	currentTime := time.Now()
+	currentTime := time.Now().UTC()
 	currentTime = currentTime.Add(1 * time.Second)
 	timeStr := currentTime.Format(time.RFC1123)
 	header.Set(rfc.IfModifiedSince, timeStr)

--- a/traffic_ops/testing/api/v3/deliveryservices_required_capabilities_test.go
+++ b/traffic_ops/testing/api/v3/deliveryservices_required_capabilities_test.go
@@ -33,7 +33,7 @@ func TestDeliveryServicesRequiredCapabilities(t *testing.T) {
 		GetTestDeliveryServicesRequiredCapabilitiesIMS(t)
 		InvalidDeliveryServicesRequiredCapabilityAddition(t)
 		GetTestDeliveryServicesRequiredCapabilities(t)
-		currentTime := time.Now().Add(-1 * time.Second)
+		currentTime := time.Now().UTC().Add(-5 * time.Second)
 		time := currentTime.Format(time.RFC1123)
 		var header http.Header
 		header = make(map[string][]string)
@@ -89,7 +89,7 @@ func GetTestDeliveryServicesRequiredCapabilitiesIMSAfterChange(t *testing.T, hea
 			}
 		})
 	}
-	currentTime := time.Now()
+	currentTime := time.Now().UTC()
 	currentTime = currentTime.Add(1 * time.Second)
 	timeStr := currentTime.Format(time.RFC1123)
 	header.Set(rfc.IfModifiedSince, timeStr)

--- a/traffic_ops/testing/api/v3/deliveryservices_test.go
+++ b/traffic_ops/testing/api/v3/deliveryservices_test.go
@@ -35,7 +35,7 @@ func TestDeliveryServices(t *testing.T) {
 	WithObjs(t, []TCObj{CDNs, Types, Tenants, Users, Parameters, Profiles, Statuses, Divisions, Regions, PhysLocations, CacheGroups, Servers, Topologies, DeliveryServices}, func() {
 		GetTestDeliveryServicesIMS(t)
 		GetAccessibleToTest(t)
-		currentTime := time.Now().Add(-1 * time.Second)
+		currentTime := time.Now().UTC().Add(-5 * time.Second)
 		time := currentTime.Format(time.RFC1123)
 		var header http.Header
 		header = make(map[string][]string)
@@ -62,7 +62,7 @@ func GetTestDeliveryServicesIMSAfterChange(t *testing.T, header http.Header) {
 	if reqInf.StatusCode != http.StatusOK {
 		t.Fatalf("Expected 200 status code, got %v", reqInf.StatusCode)
 	}
-	currentTime := time.Now()
+	currentTime := time.Now().UTC()
 	currentTime = currentTime.Add(1 * time.Second)
 	timeStr := currentTime.Format(time.RFC1123)
 	header.Set(rfc.IfModifiedSince, timeStr)

--- a/traffic_ops/testing/api/v3/divisions_test.go
+++ b/traffic_ops/testing/api/v3/divisions_test.go
@@ -29,7 +29,7 @@ func TestDivisions(t *testing.T) {
 	WithObjs(t, []TCObj{Parameters, Divisions, Regions}, func() {
 		GetTestDivisionsIMS(t)
 		TryToDeleteDivision(t)
-		currentTime := time.Now().Add(-1 * time.Second)
+		currentTime := time.Now().UTC().Add(-5 * time.Second)
 		time := currentTime.Format(time.RFC1123)
 		var header http.Header
 		header = make(map[string][]string)

--- a/traffic_ops/testing/api/v3/federation_users_test.go
+++ b/traffic_ops/testing/api/v3/federation_users_test.go
@@ -24,7 +24,7 @@ import (
 
 func TestFederationUsers(t *testing.T) {
 	WithObjs(t, []TCObj{CDNs, Types, Tenants, Users, Parameters, Profiles, Statuses, Divisions, Regions, PhysLocations, CacheGroups, Topologies, DeliveryServices, CDNFederations, FederationUsers}, func() {
-		currentTime := time.Now().Add(-1 * time.Second)
+		currentTime := time.Now().UTC().Add(-5 * time.Second)
 		time := currentTime.Format(time.RFC1123)
 		var header http.Header
 		header = make(map[string][]string)
@@ -44,7 +44,7 @@ func GetTestValidFederationIDUsersIMSAfterChange(t *testing.T, header http.Heade
 	if reqInf.StatusCode != http.StatusOK {
 		t.Fatalf("Expected 200 status code, got %v", reqInf.StatusCode)
 	}
-	currentTime := time.Now()
+	currentTime := time.Now().UTC()
 	currentTime = currentTime.Add(1 * time.Second)
 	timeStr := currentTime.Format(time.RFC1123)
 	header.Set(rfc.IfModifiedSince, timeStr)

--- a/traffic_ops/testing/api/v3/parameters_test.go
+++ b/traffic_ops/testing/api/v3/parameters_test.go
@@ -32,7 +32,7 @@ func TestParameters(t *testing.T) {
 
 	WithObjs(t, []TCObj{Parameters}, func() {
 		GetTestParametersIMS(t)
-		currentTime := time.Now().Add(-1 * time.Second)
+		currentTime := time.Now().UTC().Add(-5 * time.Second)
 		time := currentTime.Format(time.RFC1123)
 		var header http.Header
 		header = make(map[string][]string)
@@ -53,7 +53,7 @@ func GetTestParametersIMSAfterChange(t *testing.T, header http.Header) {
 			t.Fatalf("Expected 200 status code, got %v", reqInf.StatusCode)
 		}
 	}
-	currentTime := time.Now()
+	currentTime := time.Now().UTC()
 	currentTime = currentTime.Add(1 * time.Second)
 	timeStr := currentTime.Format(time.RFC1123)
 	header.Set(rfc.IfModifiedSince, timeStr)

--- a/traffic_ops/testing/api/v3/phys_locations_test.go
+++ b/traffic_ops/testing/api/v3/phys_locations_test.go
@@ -30,7 +30,7 @@ func TestPhysLocations(t *testing.T) {
 		GetTestPhysLocationsIMS(t)
 		GetDefaultSortPhysLocationsTest(t)
 		GetSortPhysLocationsTest(t)
-		currentTime := time.Now().Add(-1 * time.Second)
+		currentTime := time.Now().UTC().Add(-5 * time.Second)
 		time := currentTime.Format(time.RFC1123)
 		var header http.Header
 		header = make(map[string][]string)
@@ -69,7 +69,7 @@ func GetTestPhysLocationsIMSAfterChange(t *testing.T, header http.Header) {
 			t.Fatalf("Expected 200 status code, got %v", reqInf.StatusCode)
 		}
 	}
-	currentTime := time.Now()
+	currentTime := time.Now().UTC()
 	currentTime = currentTime.Add(1 * time.Second)
 	timeStr := currentTime.Format(time.RFC1123)
 	header.Set(rfc.IfModifiedSince, timeStr)

--- a/traffic_ops/testing/api/v3/regions_test.go
+++ b/traffic_ops/testing/api/v3/regions_test.go
@@ -28,7 +28,7 @@ import (
 func TestRegions(t *testing.T) {
 	WithObjs(t, []TCObj{Parameters, Divisions, Regions}, func() {
 		GetTestRegionsIMS(t)
-		currentTime := time.Now().Add(-1 * time.Second)
+		currentTime := time.Now().UTC().Add(-5 * time.Second)
 		time := currentTime.Format(time.RFC1123)
 		var header http.Header
 		header = make(map[string][]string)
@@ -67,7 +67,7 @@ func GetTestRegionsIMSAfterChange(t *testing.T, header http.Header) {
 			t.Fatalf("Expected 200 status code, got %v", reqInf.StatusCode)
 		}
 	}
-	currentTime := time.Now()
+	currentTime := time.Now().UTC()
 	currentTime = currentTime.Add(1 * time.Second)
 	timeStr := currentTime.Format(time.RFC1123)
 	header.Set(rfc.IfModifiedSince, timeStr)

--- a/traffic_ops/testing/api/v3/roles_test.go
+++ b/traffic_ops/testing/api/v3/roles_test.go
@@ -35,7 +35,7 @@ const (
 func TestRoles(t *testing.T) {
 	WithObjs(t, []TCObj{Roles}, func() {
 		GetTestRolesIMS(t)
-		currentTime := time.Now().Add(-1 * time.Second)
+		currentTime := time.Now().UTC().Add(-5 * time.Second)
 		time := currentTime.Format(time.RFC1123)
 		var header http.Header
 		header = make(map[string][]string)
@@ -56,7 +56,7 @@ func GetTestRolesIMSAfterChange(t *testing.T, header http.Header) {
 	if reqInf.StatusCode != http.StatusOK {
 		t.Fatalf("Expected 200 status code, got %v", reqInf.StatusCode)
 	}
-	currentTime := time.Now()
+	currentTime := time.Now().UTC()
 	currentTime = currentTime.Add(1 * time.Second)
 	timeStr := currentTime.Format(time.RFC1123)
 	header.Set(rfc.IfModifiedSince, timeStr)

--- a/traffic_ops/testing/api/v3/servers_test.go
+++ b/traffic_ops/testing/api/v3/servers_test.go
@@ -27,7 +27,7 @@ import (
 func TestServers(t *testing.T) {
 	WithObjs(t, []TCObj{CDNs, Types, Tenants, Users, Parameters, Profiles, Statuses, Divisions, Regions, PhysLocations, CacheGroups, Topologies, DeliveryServices, Servers}, func() {
 		GetTestServersIMS(t)
-		currentTime := time.Now().Add(-1 * time.Second)
+		currentTime := time.Now().UTC().Add(-5 * time.Second)
 		time := currentTime.Format(time.RFC1123)
 		var header http.Header
 		header = make(map[string][]string)
@@ -56,7 +56,7 @@ func GetTestServersIMSAfterChange(t *testing.T, header http.Header) {
 			t.Fatalf("Expected 200 status code, got %v", reqInf.StatusCode)
 		}
 	}
-	currentTime := time.Now()
+	currentTime := time.Now().UTC()
 	currentTime = currentTime.Add(1 * time.Second)
 	timeStr := currentTime.Format(time.RFC1123)
 	header.Set(rfc.IfModifiedSince, timeStr)

--- a/traffic_ops/testing/api/v3/staticdnsentries_test.go
+++ b/traffic_ops/testing/api/v3/staticdnsentries_test.go
@@ -30,7 +30,7 @@ func TestStaticDNSEntries(t *testing.T) {
 	WithObjs(t, []TCObj{CDNs, Types, Tenants, Parameters, Profiles, Statuses, Divisions, Regions, PhysLocations, CacheGroups, Servers, Topologies, DeliveryServices, StaticDNSEntries}, func() {
 		GetTestStaticDNSEntriesIMS(t)
 		GetTestStaticDNSEntries(t)
-		currentTime := time.Now().Add(-1 * time.Second)
+		currentTime := time.Now().UTC().Add(-5 * time.Second)
 		time := currentTime.Format(time.RFC1123)
 		var header http.Header
 		header = make(map[string][]string)
@@ -51,7 +51,7 @@ func GetTestStaticDNSEntriesIMSAfterChange(t *testing.T, header http.Header) {
 			t.Fatalf("Expected 200 status code, got %v", reqInf.StatusCode)
 		}
 	}
-	currentTime := time.Now()
+	currentTime := time.Now().UTC()
 	currentTime = currentTime.Add(1 * time.Second)
 	timeStr := currentTime.Format(time.RFC1123)
 	header.Set(rfc.IfModifiedSince, timeStr)

--- a/traffic_ops/testing/api/v3/statuses_test.go
+++ b/traffic_ops/testing/api/v3/statuses_test.go
@@ -27,7 +27,7 @@ import (
 func TestStatuses(t *testing.T) {
 	WithObjs(t, []TCObj{Parameters, Statuses}, func() {
 		GetTestStatusesIMS(t)
-		currentTime := time.Now().Add(-1 * time.Second)
+		currentTime := time.Now().UTC().Add(-5 * time.Second)
 		time := currentTime.Format(time.RFC1123)
 		var header http.Header
 		header = make(map[string][]string)
@@ -51,7 +51,7 @@ func GetTestStatusesIMSAfterChange(t *testing.T, header http.Header) {
 			t.Fatalf("Expected 200 status code, got %v", reqInf.StatusCode)
 		}
 	}
-	currentTime := time.Now()
+	currentTime := time.Now().UTC()
 	currentTime = currentTime.Add(1 * time.Second)
 	timeStr := currentTime.Format(time.RFC1123)
 	header.Set(rfc.IfModifiedSince, timeStr)

--- a/traffic_ops/testing/api/v3/steeringtargets_test.go
+++ b/traffic_ops/testing/api/v3/steeringtargets_test.go
@@ -32,7 +32,7 @@ func TestSteeringTargets(t *testing.T) {
 	WithObjs(t, []TCObj{CDNs, Types, Tenants, Parameters, Profiles, Statuses, Divisions, Regions, PhysLocations, CacheGroups, Servers, Topologies, DeliveryServices, Users, SteeringTargets}, func() {
 		GetTestSteeringTargetsIMS(t)
 		GetTestSteeringTargets(t)
-		currentTime := time.Now().Add(-1 * time.Second)
+		currentTime := time.Now().UTC().Add(-5 * time.Second)
 		time := currentTime.Format(time.RFC1123)
 		var header http.Header
 		header = make(map[string][]string)
@@ -58,7 +58,7 @@ func GetTestSteeringTargetsIMSAfterChange(t *testing.T, header http.Header) {
 	if reqInf.StatusCode != http.StatusOK {
 		t.Fatalf("Expected 200 status code, got %v", reqInf.StatusCode)
 	}
-	currentTime := time.Now()
+	currentTime := time.Now().UTC()
 	currentTime = currentTime.Add(1 * time.Second)
 	timeStr := currentTime.Format(time.RFC1123)
 	header.Set(rfc.IfModifiedSince, timeStr)

--- a/traffic_ops/testing/api/v3/types_test.go
+++ b/traffic_ops/testing/api/v3/types_test.go
@@ -28,7 +28,7 @@ import (
 func TestTypes(t *testing.T) {
 	WithObjs(t, []TCObj{Parameters, Types}, func() {
 		GetTestTypesIMS(t)
-		currentTime := time.Now().Add(-1 * time.Second)
+		currentTime := time.Now().UTC().Add(-5 * time.Second)
 		time := currentTime.Format(time.RFC1123)
 		var header http.Header
 		header = make(map[string][]string)
@@ -49,7 +49,7 @@ func GetTestTypesIMSAfterChange(t *testing.T, header http.Header) {
 			t.Fatalf("Expected 200 status code, got %v", reqInf.StatusCode)
 		}
 	}
-	currentTime := time.Now()
+	currentTime := time.Now().UTC()
 	currentTime = currentTime.Add(1 * time.Second)
 	timeStr := currentTime.Format(time.RFC1123)
 	header.Set(rfc.IfModifiedSince, timeStr)

--- a/traffic_ops/testing/api/v3/user_test.go
+++ b/traffic_ops/testing/api/v3/user_test.go
@@ -32,7 +32,7 @@ import (
 func TestUsers(t *testing.T) {
 	WithObjs(t, []TCObj{Tenants, Parameters, Users}, func() {
 		GetTestUsersIMS(t)
-		currentTime := time.Now().Add(-1 * time.Second)
+		currentTime := time.Now().UTC().Add(-5 * time.Second)
 		time := currentTime.Format(time.RFC1123)
 		var header http.Header
 		header = make(map[string][]string)
@@ -61,7 +61,7 @@ func GetTestUsersIMSAfterChange(t *testing.T, header http.Header) {
 	if reqInf.StatusCode != http.StatusOK {
 		t.Fatalf("Expected 200 status code, got %v", reqInf.StatusCode)
 	}
-	currentTime := time.Now()
+	currentTime := time.Now().UTC()
 	currentTime = currentTime.Add(1 * time.Second)
 	timeStr := currentTime.Format(time.RFC1123)
 	header.Set(rfc.IfModifiedSince, timeStr)

